mappings/net/minecraft/container/Container.mapping

```diff
@@ -1,13 +1,14 @@
 CLASS net/minecraft/class_71 net/minecraft/container/Container
 	FIELD field_2734 slots Ljava/util/List;
 	FIELD field_2735 currentContainerId I
-	METHOD method_2086 transferSlot (I)Lnet/minecraft/class_31;
 	METHOD method_2077 setProperty (II)V
 		ARG 1 id
 		ARG 2 value
 	METHOD method_2079 addSlot (Lnet/minecraft/class_133;)V
-	METHOD method_2087 onClosed (Lnet/minecraft/class_54;)V
 	METHOD method_2081 insertItem (Lnet/minecraft/class_31;IIZ)V
-	METHOD method_2088 onContentsChanged (Lnet/minecraft/class_134;)V
+	METHOD method_2083 getItems ()Ljava/util/List;
 	METHOD method_2084 getSlot (I)Lnet/minecraft/class_133;
+	METHOD method_2086 transferSlot (I)Lnet/minecraft/class_31;
```

Comment:
ah yes i am moving a slot ur old container will have 1 less available slot and the new one will have 1 more

Reply:
what

Reply:
I think this is shift click 
Idk

Reply:
THIS WAS ALREADY MAPPED CONC

Reply:
lmfao